### PR TITLE
update readme broken docs links

### PR DESCRIPTION
**Pull request by komichar**
Many links in this readme target the `docs.` subdomain which throws error

image

### CODE REVIEW

File: 01-Authorization-RS256/README.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Execute in command line `sh exec.sh` to run the Docker in Linux, or `.\exec.ps1`
...
-* Add authentication through more traditional **[username/password databases](https://docs.auth0.com/mysql-connection-tutorial)**.
-* Add support for **[linking different user accounts](https://docs.auth0.com/link-accounts)** with the same user.
-* Support for generating signed [Json Web Tokens](https://docs.auth0.com/jwt) to call your APIs and **flow the user identity** securely.
+* Add authentication with [multiple authentication sources](https://auth0.com/docs/authenticate/identity-providers), either social like **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce, among others**, or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider**.
+* Add authentication through more traditional **[username/password databases](https://auth0.com/docs/authenticate/database-connections/custom-db/create-db-connection#create-a-login-script)**.
```
Comment (Widcket) [inline 01-Authorization-RS256/README.md:63]:
```suggestion
* Add authentication through more traditional **[username/password databases](https://auth0.com/docs/authenticate/database-connections/custom-db/overview-custom-db-connections)**.
```

**Review by Widcket — APPROVED** [review]
LGTM

**Comment by Widcket** [comment]
cc @sergiught the CI is failing

**Comment by sergiught** [comment]
The build is failing because we don't pass secrets to builds from forked pull requests. All should be good 👍🏻 , tests will pass again once we merge to master.

cc: @Widcket

**Comment by cocojoe** [comment]
As this is just a README bump have forced the CI for now as it's unaffected anyway.

**Pull request merged**